src/steps/base/batch/default_batch_workflow_executor.ts

```diff
@@ -0,0 +1,15 @@
+import { ExecutionBindings } from '../../../workflow';
+import { BatchExecutor, BatchResult } from '../../../steps/types';
+
+export class DefaultBatchWorkflowExecutor implements BatchExecutor {
```

Comment:
SHould we call this default considering its doing custom batching logic?

Reply:
what do you mean by custom batching logic here